src/settings.rs

```diff
@@ -485,3 +488,23 @@ pub fn get_verify_config(mut cx: FunctionContext) -> JsResult<JsValue> {
         }
     }
 }
+
+/// Reset settings to their default values.
+/// This replicates the private reset() function from c2pa-rs Settings.
+pub fn reset_settings(mut cx: FunctionContext) -> JsResult<JsUndefined> {
+    // Create default settings and get their TOML representation
+    let default_settings = Settings::default();
+    let default_toml = toml::to_string(&default_settings)
```

Comment:
https://github.com/contentauth/c2pa-rs/pull/1533 may be of interest once it lands